### DISCUSSION TITLE
error looks different pdf estimates

### DISCUSSION THREAD

**Discussion by xiaoyang-mindstrong** [nlmixrdevelopment/nlmixr#622] [General]
Dear Matt, 

I have a question to see if you have encountered the problem that the error estimate in the model output pdf (showing stabilized and close to zero) is quite different from the model estimates (0.24 and SE is NA). Does the SE of the error term mean this parameter fails to converge? Appreciate your insights on this! 

best,
Xiao Yang

**Comment by mattfidler** [comment]
The `SE` estimate is not calculated `0.24`.  I would simply restart the model at the estimates to see if that changes the estimates much.

You should be able to do that with something like:

```r
fit <- nlmixr(oldFit, getData(oldFit), "focei")
```
